sdk/communication/Azure.Communication.Sms/api/Azure.Communication.Sms.netstandard2.0.cs

```diff
@@ -14,15 +14,17 @@ public partial class SmsClient
     }
     public partial class SmsClientOptions : Azure.Core.ClientOptions
     {
-        public SmsClientOptions(Azure.Communication.Sms.SmsClientOptions.ServiceVersion version = Azure.Communication.Sms.SmsClientOptions.ServiceVersion.V2021_03_07) { }
+        public SmsClientOptions(Azure.Communication.Sms.SmsClientOptions.ServiceVersion version = Azure.Communication.Sms.SmsClientOptions.ServiceVersion.V2024_02_05_Preview) { }
         public enum ServiceVersion
         {
             V2021_03_07 = 1,
+            V2024_02_05_Preview = 2,
         }
     }
     public partial class SmsSendOptions
     {
         public SmsSendOptions(bool enableDeliveryReport) { }
+        public int? DeliveryReportTimeoutInSeconds { get { throw null; } set { } }
```

Comment:
We're trying to use language built-in times for the SDK and then translate internally to the type for the REST API, consider using `TimeSpan` and rounding to seconds. You can also search through the repo to see how other Azure service SDKs handle time spans and if you find others who use numbers with a suffixed name instead, but I think the Azure .NET architects would prefer TimeSpan.